gitbootcamp.rst

```diff
@@ -203,6 +203,33 @@ from the `core-workflow  <https://github.com/python/core-workflow>`_
 repository to backport the commit.
 
 
+.. _git_from_mercurial:
+
+Applying a Patch from Mercurial to Git
+--------------------------------------
+
+Scenario:
+
+- A Mercurial patch exists but there is no pull request for it.
+
+Solution:
+
+1. Download the patch locally.
+
+2. Create a new branch and switch to it.
+
+3. For Mac and Unix::
+
+   $ patch -p1 < /path/to/issueNNNN.patch
```

Comment:
I just tried the same: parts of the patch applied with `patch`, but *not* all of it.  `git apply` refused to apply a partial patch, nothing to do with the `--git` option.  The patch is quite old (hence the conflicts), so I checked out a revision from that era via:

```
git checkout `git rev-list -n 1 --before="2011-08-22 03:25:00" master`
```

After that, `git apply` happily applied the patch, albeit while complaining about a lot of trailing whitespace.

At this point, I could get the patch updated with something like:

```
git checkout -b issue_12815
git commit -a -m "Cover smptd"
git rebase master
git mergetool
```